src/funcs.c

```diff
@@ -2825,3 +2825,71 @@ void joy_right()
   set(app->LV_GamesList, MUIA_List_Active, ind-1);
 
 }
+
+ULONG get_wb_version()
+{
+	static ULONG ver = 0UL;
+
+	if (ver != 0UL)
+	{
+		return ver;
+	}
+
+	if (WorkbenchBase == NULL)
+	{
+		//Really Workbench library should be already opened
+		// Somehow we're running without Workbench.
+		// Nothing to do since the version variable inits with 0.
+		return ver;
+	}
+
+	// Save workbench.library version for later calls
+	ver = WorkbenchBase->lib_Version;
+
+	return ver;
+}
+
+void open_current_dir()
+{
+	if (get_wb_version() < 44)
```

Comment:
You do the same check at the menu item. In workbench lower than v44 the menu item will not be shown, and there will be no other way to call this function. Am I wrong on that?

Reply:
I consider this "defence in depth" programming. The check when building the menu doesn't have any control over use of `open_current_dir()` so the function gets its own check in case there's another caller to it down the road that isn't similarly gated.

Reply:
I will agree with you on that.